### DISCUSSION TITLE
New app: Audiogridder?

### DISCUSSION THREAD

**Discussion by DengelFred** [audinux/fedora-spec#57] [Ideas]
Hello,
I'm wondering if AudioGridder (https://github.com/apohl79/audiogridder) can be added to this repository? It's a tool for running VST plugins on different hosts and operating systems over the network.
 At least the client is available for Linux, but is not yet working out of the Box on Fedora. I started already an investigation what's going on (see https://github.com/apohl79/audiogridder/discussions/1353), but unfortunately I'm not able to locate the issue by myself.
Maybe someone can have a look into it and add it finally to this repo for Fedora?
Of course, if I can help, I will do :-)

**Comment by ycollet** [comment]
Thanks for this ticket.
I will try to see if this program builds and works on Fedora.

**Reply by ycollet** [reply]
I checked the bug report, I think the problem is related to the use of a binary package downloaded from the github website.
I have a good hope that it will work once  compiled from sources :)

**Reply by ycollet** [reply]
The packages audiogridder and vst3-audiogridder are available.
Can you test these  ?

**Comment by DengelFred** [comment]
wow, that's quick :-) Thanks a lot!
I build the rpm's and installed it, works fine so far. Unfortunately the tray app is still not working, it quits after a few seconds. I will follow up  this in the audiogridder project.

**Comment by ycollet** [comment]
Ah, OK. I will try to see on the github repo if the problem already happened ...

**Comment by ycollet** [comment]
While digging the Tray code, I found an interesting command line parameter: -keeprunning
I am not able to test this yet ...

**Comment by DengelFred** [comment]
Same finding here ;-) However, the tray app is called from the plugin itself. My problem is the FlatPak and the restrictions of the format itself. I tested Bitwig & Carla as FlatPaks, and nothing works. Just installed Carla as rpm and it works like a charm :-D Unfortunately there is no other package format than flatpak for Fedora..... Now I have to investigate, how I can tweak Bitwig for using this plugin.
However, this topic is solved now. Thanks a lot for this really fast  response!
